src/config.ts

```
@@ -74,7 +95,7 @@ export const initConfig = (
     throw new Error('Configuration must contain `projectId`')
   }
 
-  if (typeof newConfig.perspective === 'string') {
+  if (typeof newConfig.perspective !== 'undefined') {
```

figured it's better if we always validate when it's provided. As it was, we would silently ignore `perspective={}` for example, which is a mistake I would appreciate being told about :)